scripts/zones/Crawlers_Nest/mobs/Demonic_Tiphia.lua

```diff
@@ -6,6 +6,16 @@ require("scripts/globals/hunts")
 -----------------------------------
 local entity = {}
 
+entity.onMobFight = function(mob, target)
+    local delay = mob:getLocalVar("delay")
+
+    -- Uses Cure V repeatedly on cooldown below 50% health
+    if os.time() > delay and mob:actionQueueEmpty() and mob:getHPP() <= 50 then
```

Comment:
https://github.com/LandSandBoat/server/pull/2098#discussion_r941761508

Reply:
Only problem with this method for Tiphia is that she casts Cure V way faster than the natural magic cooldown. Also I don't know if I understand the magic_cool mod you can put on mobs, I see no difference in how quickly they cast spells adjusting that.

Reply:
Ah ok, it maybe better to leave this for now then till we figure out wtf is going on there. that magic_cool mod should be the mobs timer between casts iirc. As of yet, we do not have support for separate timers per spell (retail does have support both a per-spell cooldown and an overall cooldown SE adjusts separately as needed but we've a surprisingly long way to go for mob spellcasting). I hadn't heard back after the other NM so I thought you missed the comment.

Reply:
Adjusted Bo'Who Warmonger to use that new follow function. Works a lot better!